### ISSUE TITLE
Some tests failling in astropy-0.3.2 under i386

### ISSUE THREAD

**Issue by sergiopasra** [astropy/astropy#2516]
Hello, I'm compilling astropy-0.3.2 to produce RPMs for Fedora. There is a test that fails in i386 but passes in x86_64. I'm getting also RuntimeWarnings in i386

The test is TestParametricModels.test_deriv_1D[model_class10]

The full log is here:
http://kojipkgs.fedoraproject.org//work/tasks/4521/6864521/build.log

Note that the testing environment is this:

```
  ============================= test session starts ==============================
  platform linux2 -- Python 2.7.6 -- py-1.4.20 -- pytest-2.5.2
  Running tests with Astropy version 0.3.2.
  Running tests in astropy.
  Platform: Linux-3.14.3-200.fc20.x86_64-i686-with-fedora-21-Rawhide
  Executable: /usr/bin/python
  Full Python Version: 
  2.7.6 (default, May  9 2014, 12:21:38) 
  [GCC 4.9.0 20140506 (Red Hat 4.9.0-3)]
  encodings: sys: ascii, locale: ANSI_X3.4-1968, filesystem: ANSI_X3.4-1968, unicode   
  bits: 20
  byteorder: little
  float info: dig: 15, mant_dig: 15
  Numpy: 1.8.1
  Scipy: 0.14.0
  Matplotlib: 1.3.1
  h5py: 2.3.0
```

**Comment by hamogu** [comment]
Is this still an issue with the 1.0 or 1.1 series?
We are not issuing bug fixes for the 0.4 series any longer, so we could close this if it's fixed in 1.0.9 and 1.1.1.

**Comment by bsipocz** [comment]
@sergiopasra - This is a rather old issue, and now we're running CI on 32bit on all supportd branches. Please feel free to reopen if you still see these failures.
